diesel_cli/src/main.rs

```diff
@@ -225,7 +225,24 @@ fn create_migrations_dir(matches: &ArgMatches) -> DatabaseResult<PathBuf> {
             .join("migrations"),
     };
 
-    if !dir.exists() {
+    if dir.exists() {
+        // This is a cleanup code for migrating from an
+        // older version of diesel_cli that set a `.gitkeep` instead of a `.keep` file.
+        // TODO: remove this after a few releases
+        if let Some(dir_entry) = fs::read_dir(&dir)
+            .unwrap()
+            .filter(|x| x.is_ok())
```

Comment:
This should probably use `.filter_map` instead.

Reply:
I've fixed this!